.env.example

```diff
@@ -49,4 +49,5 @@ TWILIO_MESSAGE_VALIDITY_PERIOD=
 DST_REFERENCE_TIMEZONE='America/New_York'
 PASSPORT_STRATEGY=local
 EXPERIMENTAL_TAGS=1
-TEXTER_SIDEBOXES=contact-reference,tag-contact
\ No newline at end of file
+TEXTER_SIDEBOXES=contact-reference,tag-contact
+MULTI_TENANT=true
```

Comment:
Every Spoke instance is multi-tenant, so I don't think we need a variable for this.  Please remove.

Reply:
done!